aten/src/ATen/native/Activation.cpp

```diff
@@ -599,10 +599,12 @@ Tensor rrelu_with_noise_backward(
 }
 
 Tensor rrelu(const Tensor & self, const Scalar& lower, const Scalar& upper, bool training, c10::optional<Generator> generator) {
+  TORCH_CHECK(lower.to<double>() < upper.to<double>(), "Lower bound must be less than the upper bound")
```

Comment:
Why not weakly less? `uniform_()` allows the lower and upper bounds of the distribution to be the same

Reply:
Hi, @mruberry! Thanks for pointing it out. This makes sense to me! Have updated it.